graphql_compiler/schema_transformation/merge_schemas.py

```diff
@@ -20,19 +21,41 @@
 )
 
 
-def merge_schemas(schema_id_to_ast):
-    """Check that input schemas do not contain conflicting definitions, then merge.
+CrossSchemaEdgeDescriptor = namedtuple(
+    'CrossSchemaEdgeDescriptor', (
+        'edge_name',  # str, name used for the corresponding in and out fields
+        'outbound_side',  # FieldReference for the outbound (source) field
+        'inbound_side',  # FieldReference for the inbound (sink) field
+        'out_edge_only',  # bool, defaults to False, whether to not add the in-edge
+    )
+)
+
+
+CrossSchemaEdgeDescriptor.__new__.__defaults__ = (False,)
+
+
+FieldReference = namedtuple(
+    'FieldReference', (
+        'schema_id',  # str, identifier for the schema of the field
+        'type_name',  # str, name of the type of the field
```

Comment:
nit: How about `vertex_name`. `type_name` seems to vague to me.

Reply:
There are a lot of places in this file where `type` or `type_name` is used, and `vertex` isn't used anywhere yet. I've considered changing to using the graphql compiler's flavor of terminology, but I think a lot of variable names and documentation would need to be changed together. I'll work on this after resolving more immediate concerns.

Reply:
I will change the comment here for clarify for now.